### PR TITLE
Add Submariner UI for Openstack cloud

### PR DESCRIPTION
**Pull request by aswinsuryan**
* Added text boxes to accept OpenStack credentials and instance type
 * The submariner config will be created with the values provided

Fixes: [backlog#19297](https://github.com/stolostron/backlog/issues/19297)

Signed-off-by: Aswin Suryanarayanan <aswinsuryan@gmail.com>

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@aswinsuryan Looks good to me. I just placed a hold so that one of your colleagues can review for the Submariner business logic. Feel free to remove the hold once you have a 2nd reviewer. Thanks.

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2416,6 +2416,7 @@
     "The name of the vSphere cluster to use.": "The name of the vSphere cluster to use.",
     "The name of the vSphere datacenter to use.": "The name of the vSphere datacenter to use.",
     "The OpenStack clouds.yaml file, including the password, to connect to the OpenStack server.": "The OpenStack clouds.yaml file, including the password, to connect to the OpenStack server.",
+    "The OpenStack instance type of the gateway node that will be created on the managed cluster (default PnTAE.CPU_16_Memory_32768_Disk_80).": "The OpenStack instance type of the gateway node that will be created on the managed cluster (default PnTAE.CPU_16_Memory_32768_Disk_80).",
```
Comment (sridhargaddam) [inline frontend/public/locales/en/translation.json:2419]: When we run cloud prepare using subctl, the default instanceType is PnTAE.CPU_4_Memory_8192_Disk_50, which does not match with this change. Lets use consistent values across subctl and ACM.

https://github.com/submariner-io/subctl/blob/95a3edd76b3e75b35453941021e10ab468b80b37/cmd/subctl/rhos.go#L78

Comment (aswinsuryan) [reply frontend/public/locales/en/translation.json]: done.

**Comment by aswinsuryan** [comment]
@KevinFCormier Thanks for your review. I am getting an alert from `Splunk Cloud` about a "Potential data leak" since the test has secrets at [1] and since it is a false positive asks me to add a  '.gitleaks.toml' in the root.  Is that the procedure followed in this repo.


[1]https://github.com/aswinsuryan/console/blob/041a2007de699a82a6366d0aa774b475fec7d0da/frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetDetails.test.tsx#L392

**Comment by KevinFCormier** [comment]
@aswinsuryan I didn't think this was active yet, but I think that is the correct course of action. @pshickeydev, can you confirm? Patrick outlined how we can have this same scanning done in a git pre-commit hook in a recent security guild meeting, but I have not set that up yet.

**Review by vthapar — APPROVED** [review]

**Comment by aswinsuryan** [comment]
/remove hold

@KevinFCormier  after adding` //gitleaks:allow` in the line that had the issue, I stopped getting the warning about the leak. Removing the hold

**Pull request merged**